ZelBack/src/services/syncthingService.js

```diff
@@ -96,11 +99,258 @@ async function systemPing(req, res) {
   return res ? res.json(response) : response;
 }
 
+async function getConfigRestartRequired(req, res) {
+  const response = await performRequest('get', '/rest/config/restart-required');
+  return res ? res.json(response) : response;
+}
+
 async function getConfigFolders(req, res) {
-  const response = await performRequest('get', '/rest/config/folders');
+  let { id } = req.params;
+  id = id || req.query.id;
```

Comment:
perhaps using ?? what if id is 0?

Reply:
if id is 0 or not provided, '/rest/config/folders' will be called. syncthing rest api accepts both forms. if id is not passed it will return an array of folders.